single-audit/resolve_findings/templates/resolve_findings/list.html

```diff
@@ -0,0 +1,17 @@
+<!-- @todo: ADD USWDS CLASSES! -->
```

Comment:
Is this `list.html` page ultimately going to grow into the Findings Overview page? Or is it more of an admin view into findings?

Reply:
@alexsoble It's intended to be the Finding Overview page. The admin view into findings is built-in Django functionality accessible via `/admin`. (Which could be customized at some point if desired.)